File: test_scripts/test1/lint_issues.R
# Script with linting issues but working code

source("utils.R")

# Bad style - long line and spacing issues
very_long_variable_name_that_exceeds_normal_limits<-function(x,y,z){return(x+y+z)}

# Trailing whitespace and inconsistent spacing
data<-c(1,2,3,4,5)   
result =   mean( data )  

# This works but has style issues
print(result)

File: test_scripts/python/test1/lint_issues.py
"""Script with lint issues but works - depends on utils.py"""

import utils

# Long variable name (lint warning)
this_is_a_very_long_variable_name_that_exceeds_normal_conventions = 3

# Multiple statements on one line (lint warning)
x = 1; y = 2; z = 3

# Unused variable (lint warning)
unused_variable = "not used"

result = this_is_a_very_long_variable_name_that_exceeds_normal_conventions
print(result)